Makefile

```diff
@@ -18,10 +18,4 @@ test-all:
 	mix test
 	cd interop && mix run script/run.exs
 
-# This is heavy
-ci-cron:
-	cd interop && mix deps.get && mix run script/run.exs --rounds 1000 --concurrency 30 && cd -
-	mix deps.get && bash .ci/build-plt-cache.sh && mix dialyzer
```

Comment:
We may have a ci job to run dialyzer every commit that merge to master branch?

Reply:
Yeah, we can add a cached dialyzer step :)